### PR TITLE
Added French

### PR DESCRIPTION
**Pull request by lipis**
Soon.. https://github.com/wireapp/wire-desktop/pull/39

We need more translators: https://crowdin.com/project/wire-cs/fr :)
(no need to translating the website)

### CODE REVIEW

**Comment by gregor** [comment]
We need to check the German translations before the merge. Some things got lost in translation with this commit.

**Comment by gregor** [comment]
@herrmannplatz Do you still have the script to check for obsolete strings? I think we have accumulated a few by now and should clean up decrease effort required to translate into new languages.

File: app/script/localization/strings-de.coffee [hunk 1]
```diff
@@ -517,7 +518,7 @@ z.string.de.system_notification_member_leave_left = '%s.first_name hat die Unter
 z.string.de.system_notification_member_leave_removed_many = '%s.first_name hat %no Kontakte aus der Unterhaltung entfernt'
 z.string.de.system_notification_member_leave_removed_one = '%s.first_name hat %@.first_name aus der Unterhaltung entfernt'
 z.string.de.system_notification_ping = 'Hat gepingt'
-z.string.system_notification_reaction = '%reaction deine Nachricht'
```
Comment (lipis) [inline app/script/localization/strings-de.coffee:520]: That's a clue here.. `.de` is missing.. :D

**Comment by lipis** [comment]
Something is always going on with the german translation.. I'll remove the german changes anyways from this PR.. but can't figure out why it's happening :)

**Comment by gregor** [comment]
Yes, I still cannot push to crowdin. Still haven't set it up. Manual error on my side.

**Comment by lipis** [comment]
New PR coming soon

**Pull request closed without merging**